src/Atmos/Model/ref_state.jl

```diff
@@ -169,3 +170,40 @@ function (profile::LinearTemperatureProfile)(
     end
     return (T, p)
 end
+
```

Comment:
Remove `LinearTemperatureProfile` above and write out `DryAdiabaticProfile` explicitly, without reference to linear profile. See design doc.

Reply:
We may want to make this a separate PR, revamping the temperature profiles, because we will need to make sure all other experiments still run after that.

Reply:
I created an issue.